lib/utils.py

```diff
@@ -92,3 +92,40 @@ def inline_anova(anova,
 		if pythontex_safe:
 			inline = inline.replace("\\","\\\\")
 	return inline
+
+def bold_max_row_value(df, df_tex: str):
+    """
+    Wraps the maximum value for each row with \textbf{} in a df.to_latex() string.
```

Comment:
Perhaps make the explanation clearer. i.e. detect numeric rows in a pandas dataframe LaTeX string (as produced by `df.to_latex()`), and highlight the largest value by wrapping it with a LaTeX bold tag (`\textbf{}`). Maybe even longer than that. Had to read the function to understand what it does :P

Reply:
I agree, not very clear^^